app/src/main/res/layout/habit_row.xml

```diff
@@ -26,12 +26,34 @@
                 android:layout_height="wrap_content"
                 android:text="@string/habit_title"
                 android:textAlignment="center"
-                android:textSize="24sp"
                 android:textColor="@color/habit_list_text"
+                android:textSize="24sp"
                 android:textStyle="bold"
                 app:layout_constraintBottom_toBottomOf="parent"
                 app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.395"
                 app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.542" />
```

Comment:
What are these hardcoded numbers for? This probably won't scale well on different screen sizes.